sw/device/tests/sim_dv/data_integrity_escalation_reset_test.c

```diff
@@ -148,6 +255,20 @@ void ottf_external_isr(void) {
     CHECK(irq_id == kTopEarlgreyPlicIrqIdAlertHandlerClassa,
           "Unexpected irq_id, expected %d, got %d",
           kTopEarlgreyPlicIrqIdAlertHandlerClassa, irq_id);
+    LOG_INFO("Got expected alert handler interrupt %d", irq_id);
+
+    // Disable these interrupts from alert_handler so they don't keep happening
+    // until NMI.
+    uint32_t irq =
+        (irq_id -
+         (dif_rv_plic_irq_id_t)kTopEarlgreyPlicIrqIdAlertHandlerClassa);
+    CHECK_DIF_OK(dif_alert_handler_irq_set_enabled(&alert_handler, irq,
+                                                   kDifToggleDisabled));
+
+    // Disable this interrupt to prevent it from continuously firing. This
+    // should not prevent escalation from continuing.
+    CHECK_DIF_OK(dif_rv_plic_irq_set_enabled(&plic, irq_id, kPlicTarget,
```

Comment:
Huh? You said disable, but apparently you are enabling it.

Reply:
Fair - although I think it worked because we also disable it on the alert handler side in the function just above this. I will update this.